documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_3_Governance_Perspective/4_3_X_Goverance_for_Vocabularies.md

```diff
@@ -0,0 +1,29 @@
+
+### Governance for Vocabularies ###
```

Comment:
- Short draft to get a taste, some things missing
   - [ ] Why shared vocabularies need a governance
   - [ ] Problem description: reliability of vocabularies, quality assurance
   - [ ] BOMOS (added adoption because BOMOS is a holistic approach)
   - [ ] Link to DILIGENT, happy  to oblige in next iteration
   - [ ] Other good practice, naming conventions, (self-explanatory) documentation
   - [ ] Change requests life cycle
   - [ ] Versioning
   - [ ] Zooming in on the process of actual creating/updating a shared ontology
   - [ ] Explicit reference to section `Vocabulary Intermediary` in chapter 3.1, where governance and SDO are mentioned briefly. This and 3.1 align nicely at the moment.